prow/oss/config.yaml

```diff
@@ -23,6 +23,7 @@ tide:
     GoogleCloudPlatform/osconfig: squash
     GoogleCloudPlatform/artifact-registry-apt-transport: squash
     GoogleCloudPlatform/artifact-registry-yum-plugin: squash
+    GoogleCloudPlatform/blueprints: squash
```

Comment:
FYI this doesn't actually do anything unless you enable the merge automation by modifying the queries below. Thats for the best though. You shouldn't add merge automation until you have presubmits set up.